src/components/cesium_map/api/spatial.js

```diff
@@ -545,11 +560,43 @@ export class ISamplesSpatial {
     document.body.removeChild(dummy);
   }
 
-  copyPrimitiveId(movement) {
+  openLinkInNewWindow(URL) {
+    window.open(URL, "_blank");
```

Comment:
Does this work now?  I remember it was having problems last week

Reply:
Yeah, I forgot to mention it this morning. It works right now and I also add a function to handle the unknown `producedBy_resultTime`. But we don't  need `openLinkInNewWindow` function. I will delete it.